app/models/room.rb

```diff
@@ -1,3 +1,10 @@
 class Room < ApplicationRecord
   belongs_to :hotel
+  has_many :guest_rooms
+  has_many :guests, through: :guest_rooms
+
+  def count_guests
+    self.guests.count
```

Comment:
Nice encapsulation here! It wasn't strictly called out in the US so it's not required but it's nice to do a `distinct` here before the count to make sure there aren't any guests duplicated in case they reserved multiple times